packages/ignite/src/templates/plugin/templates/Example.js.ejs

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react'
+import { View, Text } from 'react-native'
+import ExamplesRegistry from '../../../App/Services/ExamplesRegistry'
```

Comment:
@skellock - we should really see about adding that `~/App` plugin by default so these become readable.